packages/@okta/vuepress-site/docs/reference/architecture-center/directory-coexistence/overview/index.md

```diff
@@ -0,0 +1,103 @@
+2. How do you find and merge duplicate user profiles?
+3. Can you update your applications to use OpenID Connect (OIDC)?
+
+By considering these, you can also start understanding how long it may take to apply this architecture to your network.
+
+### Just-in-time Migration
+
+Until you are ready to use only Okta for authentication, your users can sign into Okta, and Okta will delegate authentication to your current identity provider. Your current identity directories and Universal Directory may coexist as long as required.
+
+There are three strategies for user migration:
+
+1. In a **bulk migration** strategy, you divide users, groups, and data into sections. Create an order in which each section will be migrated as a whole, and a schedule for migrating each section. For example, you may migrate everything in one go or create twelve portions and migrate one a month.
+2. In a **just-in-time strategy**, when a user logs into their application, their credentials are checked against their original identity provider. If the user exists there but not in Okta, the user is mirrored in Okta. If the user already exists in the original provider and Okta, and the information in the original has been updated, then the user's information in Okta is updated from the original.
+3. A hybrid of bulk and just-in-time strategies. For example, you could consider migrating active users using a just-in-time strategy and inactive users in bulk in one go.
+
+> **Note:** A Just-in-time strategy is supported when migrating from Azure Active Directory or an on-prem LDAP directory server.
+
+### De-duplicating user profiles
+
+A common issue when migrating user profiles into a common directory from multiple sources is how to identify and merge several profiles representing the same user into one.
+
+When you import users, you can create Okta rules to match any attribute currently mapped from a source user profile to an Okta user profile. Attribute matching helps you sync identities from multiple applications and determine whether an imported user is new or if the user profile already exists in Okta. For more information on how to create matches, see [Match imported user attributes](https://help.okta.com/okta_help.htm?type=oie&id=ext_match-user).
+
+### Updating applications to OpenID Connect
+
+OAuth 2.0 and OpenID Connect (OIDC) are the standard protocols on which Okta's authentication and authorization solutions are based.
+
+Using Universal Directory as a single identity provider for your applications, you can already take advantage of [Okta Workflows](https://www.okta.com/platform/workflows/) to simplify onboarding and offboarding, user management, and reporting. To take full advantage of Okta's features and enable [Single Sign-On](https://www.okta.com/topic/single-sign-on/), multifactor authentication, passwordless access, and more, all your applications should use OIDC to authenticate users rather than SAML or another solution.
```

Comment:
Maybe reference help.okta.com for Workflows: https://help.okta.com/wf/en-us/Content/Topics/Workflows/workflows-main.htm

Reply:
changed